app/src/main/java/com/sunasterisk/moviedb_51/utils/BindingUtils.kt

```diff
@@ -0,0 +1,81 @@
+package com.sunasterisk.moviedb_51.utils
+
+import android.view.LayoutInflater
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import com.google.android.material.chip.Chip
+import com.google.android.material.chip.ChipGroup
+import com.sunasterisk.moviedb_51.R
+
+@BindingAdapter("bindImageGenres")
+fun ImageView.bindImageGenres(genresID: Int?) {
+    genresID ?: return
+    this.setImageResource(
+        when (genresID) {
+            GenresTypes.ACTION.genreId -> GenresTypes.ACTION.genreImageResource
+            GenresTypes.ADVENTURE.genreId -> GenresTypes.ADVENTURE.genreImageResource
+            GenresTypes.ANIMATION.genreId -> GenresTypes.ANIMATION.genreImageResource
+            GenresTypes.COMEDY.genreId -> GenresTypes.COMEDY.genreImageResource
+            GenresTypes.CRIME.genreId -> GenresTypes.CRIME.genreImageResource
+            GenresTypes.DOCUMENTARY.genreId -> GenresTypes.DOCUMENTARY.genreImageResource
+            GenresTypes.DRAMA.genreId -> GenresTypes.DRAMA.genreImageResource
+            GenresTypes.FAMILY.genreId -> GenresTypes.FAMILY.genreImageResource
+            GenresTypes.FANTASY.genreId -> GenresTypes.FANTASY.genreImageResource
+            GenresTypes.HISTORY.genreId -> GenresTypes.HISTORY.genreImageResource
+            GenresTypes.HORROR.genreId -> GenresTypes.HORROR.genreImageResource
+            GenresTypes.MUSIC.genreId -> GenresTypes.MUSIC.genreImageResource
+            GenresTypes.MYSTERY.genreId -> GenresTypes.MYSTERY.genreImageResource
+            GenresTypes.ROMANCE.genreId -> GenresTypes.ROMANCE.genreImageResource
+            GenresTypes.SCIENCE_FICTION.genreId -> GenresTypes.SCIENCE_FICTION.genreImageResource
+            GenresTypes.TV_MOVIE.genreId -> GenresTypes.TV_MOVIE.genreImageResource
+            GenresTypes.THRILLER.genreId -> GenresTypes.THRILLER.genreImageResource
+            GenresTypes.WAR.genreId -> GenresTypes.WAR.genreImageResource
+            GenresTypes.WESTERN.genreId -> GenresTypes.WESTERN.genreImageResource
+            else -> GenresTypes.NONE.genreImageResource
+        }
+    )
+}
+
+@BindingAdapter("bindImage")
+fun ImageView.loadImageUrl(
+    image_path: String?
+) {
```

Comment:
```suggestion
fun ImageView.loadImageUrl(imagePath: String?) {
```
Nếu imagePath null thì sao ?